test/Negative.test.js

```diff
@@ -1,16 +1,15 @@
-const assert = require('assert');
 const math = require('mathjs');
 
 const flatten = require('../lib/util/flattenOperands');
 const print = require('../lib/util/print');
 
 const Negative = require('../lib/Negative');
 
+const TestUtil = require('./TestUtil');
+
 function testNegate(exprString, outputStr) {
-  it(exprString + ' -> ' + outputStr, () => {
-    assert.deepEqual(
-      print(Negative.negate(flatten(math.parse(exprString)))),outputStr);
-  });
+  const inputStr = Negative.negate(flatten(math.parse(exprString)));
```

Comment:
ohh right sorry I forgot that negate doesn't return a nodestatus... darn. maybe it's clearest if we just keep it the way it originally was then